Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -274,6 +430,11 @@ scoped notation "ℵ_ " => aleph
 /-- `ℵ₁` is the first uncountable ordinal. -/
```

Comment:
```suggestion
/-- `ℵ₁` is the first uncountable cardinal. -/
```
(sorry, you didn't touch this in this PR, but this seems like it should be fixed)